compiler/src/iree/compiler/Codegen/TransformStrategies/Common/Common.cpp

```diff
@@ -284,7 +284,9 @@ Value mlir::iree_compiler::buildPad(
 // TODO: configure patterns.
 Value mlir::iree_compiler::buildVectorize(ImplicitLocOpBuilder &b, Value funcH,
                                           bool applyCleanups) {
-  funcH = b.create<VectorizeOp>(funcH);
+  funcH = b.create<VectorizeOp>(funcH,
+                                /*vectorizePadding=*/false,
+                                /*vectorizeNdExtract=*/true);
```

Comment:
I set this as a blanket default for now but can make it toggle-able if are implications for other strategies.